src/main/java/org/gradlex/javaecosystem/capabilities/rules/jakarta/JavaxTransactionApiRule.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright the GradleX team.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradlex.javaecosystem.capabilities.rules.jakarta;
+
+import org.gradle.api.artifacts.CacheableRule;
+import org.gradle.api.artifacts.ModuleVersionIdentifier;
+import org.gradlex.javaecosystem.capabilities.rules.CapabilityDefinitions;
+import org.gradlex.javaecosystem.capabilities.rules.CapabilityDefinitionsRule;
+import org.gradlex.javaecosystem.capabilities.util.VersionNumber;
+
+import javax.inject.Inject;
+
+@CacheableRule
+public abstract class JavaxTransactionApiRule extends CapabilityDefinitionsRule {
+
+    static final String FIRST_JAKARTA_VERSION = "1.3.1";
```

Comment:
This should be `2.0.0`.

It means that this is the first version with and `jakarta` package inside (which won't be a conflict with `javax`)
`1.x` versions contain the `javax` package inside.